servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -189,7 +189,31 @@ public InodeDirectory getRoot() {
   public CreatePathResult createPath(TachyonURI path, long blockSizeBytes, boolean recursive,
       boolean directory)
           throws FileAlreadyExistException, BlockInfoException, InvalidPathException {
-    return createPath(path, blockSizeBytes, recursive, directory, System.currentTimeMillis());
+    return createPath(path, blockSizeBytes, recursive, directory, System.currentTimeMillis(),
+                      Constants.NO_TTL);
+  }
+
+  /**
+   * Creates a file or directory at the given path.
+   *
+   * @param path the path
+   * @param blockSizeBytes block size in bytes, if it is to create a file, the blockSizeBytes should
+   *        not be fewer than 1, otherwise, it is ignored, can be set to 0
+   * @param recursive if it is true, create any necessary but nonexistent parent directories of the
+   *        path, otherwise, throw InvalidPathException if there some necessary parent directories
+   *        is nonexistent
+   * @param ttl ttl for file expiration
+   * @param directory if it is true, create a directory, otherwise, create a file
+   * @return a {@link CreatePathResult} representing the modified inodes and created inodes during
+   *         path creation.
+   * @throws FileAlreadyExistException
+   * @throws BlockInfoException
+   * @throws InvalidPathException
+   */
+  public CreatePathResult createPath(TachyonURI path, long blockSizeBytes, boolean recursive,
+                                     boolean directory, long ttl)
```

Comment:
The indentation here is off (applies to other places too). We use a four space indent on a continuing line.